nnpdf_data/nnpdf_data/commondata/ATLAS_Z0_7TEV_46FB/metadata.yaml

```diff
@@ -59,49 +59,51 @@ implemented_observables:
       - $Z_{high}$
       - $Z_{high}$
       - $Z_{high}$
-    plot_x: k1
+    plot_x: abs_eta
   kinematic_coverage:
-  - k1
-  - k2
-  - k3
+  - abs_eta
+  - m_ll2
+  - sqrts
   kinematics:
     variables:
-      k1:
-        description: Variable k1
-        label: k1
+      abs_eta:
+        description: Absolute dilepton rapidity
+        label: abs_eta
         units: ''
-      k2:
-        description: Variable k2
-        label: k2
-        units: ''
-      k3:
-        description: Variable k3
-        label: k3
-        units: ''
-    file: kinematics_CC-Y.yaml
+      m_ll2:
+        description: Squared dilepton mass
+        label: m_ll2
+        units: 'GeV^2'
+      sqrts:
+        description: Square root of center of mass energy
+        label: sqrts
+        units: 'GeV'
+    file: kinematics_cc.yaml
   theory:
     conversion_factor: 1000.0
     operation: 'null'
     FK_tables:
     - - ATLAS_DY_7TEV_CC
-  data_uncertainties: []
+  data_uncertainties: [uncertainties_cc.yaml]
   variants:
     legacy:
       data_uncertainties:
       - uncertainties_legacy_CC-Y.yaml
-  data_central: data_legacy_CC-Y.yaml
+      data_central:
```

Comment:
I noticed that if you remove data_central from `variants: legacy` the tests pass again

Reply:
Ok, I will just check if changing the formatting of this helps

Reply:
The tests pass now, excellent!